how-to/how-to-wp-install-aet.md

```diff
@@ -0,0 +1,52 @@
+---
+title: How to Install the Telerik Application Enrollment Token
+description: Install the Telerik AET on WP device and add the Telerik AD workplace.
+type: how-to
+page_title: Adding the Telerik AD Workplace on WP8 Devices
+slug: how-to-install-wp-aet
+position:
+tags: appbuilder, windows-phone, deployment
+teampulseid: 
+ticketid:
+pitsid:
+
+---
+                    
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Cloud Build - Application package, Deployment - QR code</td>	
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>	
+  </tr>
+  <tr>
+    <td>Mobile OS</td>
+    <td>Windows Phone</td>	
+  </tr> 
+</table>
+
+## Description
+To deploy Windows Phone 8 application packages via QR code, you need to have the Telerik Application Enrollment Token (AET) installed on the device. It configures the Telerik AD workplace account, which is essential for installing and running AppBuilder apps on test devices.
```

Comment:
I think that "Windows Phone 8 apps built with {{site.tp}}"" or similar sounds better. Anyways, please use the variable for the respective product or service.